### PR TITLE
feat(manifest): Adds location permissions to leanplum-location Androi…

### PR DESCRIPTION
**Pull request by Enuviel**
feat(manifest): Adds location permissions to leanplum-location AndroidManifest.xml.

### CODE REVIEW

**Comment by Enuviel** [comment]
test this please

File: AndroidSDKGcm/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -2,12 +2,13 @@
...
 
     <uses-sdk android:minSdkVersion="14"/>
-    <!-- Permissions for GCM -->
+    <!-- Permissions for GCM. -->
     <uses-permission android:name="com.google.android.c2dm.permission.RECEIVE"/>
     <permission
         android:name="${applicationId}.permission.C2D_MESSAGE"
         android:protectionLevel="signature"/>
     <uses-permission android:name="${applicationId}.permission.C2D_MESSAGE"/>
+
```
Comment (alexisoyama) [inline AndroidSDKGcm/src/main/AndroidManifest.xml]: empty line

File: AndroidSDKLocation/src/main/AndroidManifest.xml [hunk 2]
```diff
@@ -2,6 +2,10 @@
           xmlns:android="http://schemas.android.com/apk/res/android">
 
     <uses-sdk android:minSdkVersion="14"/>
+    <!-- Permissions for geofencing. -->
+    <uses-permission android:name="android.permission.ACCESS_FINE_LOCATION"/>
+    <uses-permission android:name="android.permission.ACCESS_COARSE_LOCATION"/>
```
Comment (alexisoyama) [inline AndroidSDKLocation/src/main/AndroidManifest.xml]: It seems like you only need ACCESS_FINE_LOCATION  or ACCESS_COARSE_LOCATION  by this https://developer.android.com/guide/topics/location/strategies.html#Permission

Comment (Enuviel) [reply AndroidSDKLocation/src/main/AndroidManifest.xml]: there were some cases when you need both, I prefer to keep both.

File: AndroidSDKGcm/src/main/AndroidManifest.xml (hunk 1 above)
Comment (Enuviel) [reply AndroidSDKGcm/src/main/AndroidManifest.xml]: That was planned empty line. It will separate permissions and application.

**Comment by alexisoyama** [comment]
Cool. Makes sense. Approved.

**Comment by Enuviel** [comment]
test this please

**Pull request merged**